src/tools/metric_name.h

```diff
@@ -37,7 +37,7 @@ const char kCurveVersionMetricName[] = "curve_version";
 
 // snapshot clone server metric name
 const char kSnapshotCloneConfMetricName[] =
```

Comment:
It will not be affected. This is the tool to obtain the indicator information based on the metric name. Now it cannot be obtained. After checking, the name is wrong. I don’t know when the metric name was changed, but the tool has not changed.